maldoca/base/file.cc

```diff
@@ -46,6 +45,10 @@
 #include "re2/re2.h"
 #endif  // MALDOCA_CHROME
 
+// Key to xor-decode testdata files as a safety measure and to e.g. avoid being
+// deleted by AVs.
+constexpr int32_t kXorKey = 0x42;
```

Comment:
shouldn't this be uchar or uint8_t instead of int32_t?

Reply:
done